### PR TITLE
Update categories and tags

### PR DESCRIPTION
**Pull request by petecheslock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,30 +23,48 @@
   "main": "./out/extension.js",
   "categories": [
     "Visualization",
-    "Debuggers",
-    "Testing"
+    "Machine Learning",
+    "Programming Languages",
+    "Snippets"
```
Comment (kgilpin) [inline package.json:28]: Someone said just 3 categories?

File: package.json [hunk 2]
```diff
@@ -23,30 +23,48 @@
...
+    "javascript",
+    "map",
+    "mistral",
+    "mixtral",
+    "monitoring",
+    "node",
+    "node.js",
     "observability",
+    "ollama",
+    "openai",
```
Comment (kgilpin) [inline package.json:55]: gpt chatgpt

File: package.json [hunk 3]
```diff
@@ -23,30 +23,48 @@
...
+    "code completion",
+    "code explanation",
+    "code recommendation",
+    "coding assistant",
+    "dependency",
     "diagram",
     "diff",
-    "software design",
-    "visualization",
+    "documentation",
```
Comment (kgilpin) [inline package.json:44]: llm

File: package.json (hunk 1 above)
Comment (petecheslock) [reply package.json:28]: Looking at competitors some are using 4 which seems to work! 
Screenshot 2024-05-01 at 11 13 50 AM

**Pull request merged**